addons/payment_asiapay/__init__.py

```diff
@@ -0,0 +1,8 @@
+from . import models
```

Comment:
Please include this line at the beginning of every Python file. It's just a safety net, should the file be displayed somewhere outside Odoo's repositories without context.
```suggestion
# Part of Odoo. See LICENSE file for full copyright and licensing details.

from . import models
```